task/reindex.go

```diff
@@ -383,6 +383,16 @@ func transformZebedeeDoc(ctx context.Context, tracker *Tracker, errChan chan err
 			tracker.Inc("untransformed-notitle")
 			continue // move on to the next extracted doc
 		}
+
+		if !isEditorialSeries(zebedeeData.DataType) && zebedeeData.Description.MigrationLink != "" {
+			// Do not index if not editorial series and has migrationLink
+			log.Info(ctx, "not indexing content as it has been migated", log.Data{
```

Comment:
Not sure about the wording...

```suggestion
			log.Info(ctx, "content migrated - not indexing", log.Data{
```

Reply:
Done